File: backend/entity/rating.go
package entity

import "gorm.io/gorm"

type Rating struct {
    gorm.Model

    Score        uint   `json:"score"` 
    Comment      string `json:"comment"`

    BookerUserID uint   `json:"booker_user_id"`
    BookerUser   Users  `gorm:"foreignKey:BookerUserID;references:ID"`

    PosterUserID uint   `json:"poster_user_id"`
    PosterUser   Users  `gorm:"foreignKey:PosterUserID;references:ID"`
}

File: frontend/src/interfaces/Rating.ts
export interface RatingInterface {
    id: number; // ID ของการให้คะแนน
    score: number; // คะแนน (เช่น 1-5)
    comment?: string; // คอมเมนต์เกี่ยวกับการให้คะแนน
    poster_user_id: number;
    booker_user_id: number;
}